SubmittingPatches-backports.rst

```diff
@@ -0,0 +1,344 @@
+https://github.com/ceph/ceph.
+
+Before you open that PR, please read (at the very least) the following general
+rules::
+
+Backports should state (either in the backport PR or in the tracker issue):
+
+1. what bug is fixed
+2. why this fix is the minimal way to do it
+3. why does this need to be fixed in <release>
+
+The following rules apply to the actual implementation of the backport:
+
+* all bugfixes and features should land in master first
+* commits to stable branches should be cherry-picked from master
+* cherry-picks must be done using ``git cherry-pick -x``
+* if a commit cannot be cherry-picked from master, the commit message must explain why that was not possible
+* the commit message generated by ``git cherry-pick -x`` must not be modified, except to add a "Conflicts" section
+* the "Conflicts" section must mention all files where changes had to made manually (not just conflicts flagged by git)
+* the "Conflicts" section should also describe the manual changes that were made
+* if a change is to be backported to multiple stable branches, a tracker issue is needed
+
+For more information on that last point, see the `Tracker workflow`_ section.
+
+.. contents::
+   :depth: 3
+
+
+Tracker workflow
+----------------
+
+Any change that is to be backported to multiple stable branches should have
+a tracker issue associated with it. If there isn't one yet, open one at
+https://tracker.ceph.com. This issue should be in the "Bug" or "Feature"
+trackers of the relevant subproject under the "Ceph" parent project (or
+in the "Ceph" project itself if none of the subprojects are a good fit).
+The stable branches to which the master changes are to be cherry-picked should
+be listed in the "Backport" field. For example::
+
+    Backport: mimic, nautilus
+
+Once the PR targeting master is open, add the PR number assigned by GitHub to
+the tracker issue. For example, if the PR number is 99999::
+
+    Pull request ID: 99999
+
+If you do not have sufficient permissions to modify any field of the tracker
+issue, just add a comment describing what changes you would like to make.
+Someone with permissions will make the necessary modifications on your behalf.
+
+Once the master PR has been merged, after checking that the change really needs
+to be backported and the Backport field has been populated, make sure the
+tracker issue's Status field gets changed to "Pending Backport".
+
+For straightforward backports, that's all that you (as the developer of the fix)
+need to do. Volunteers from the `Stable Releases and Backports team`_ will
+proceed to create Backport issues to track the necessary backports and stage the
+backports by opening GitHub PRs with the cherry-picks. If you don't want to
+wait, and provided you have sufficient permissions at https://tracker.ceph.com,
+you can open the Backport tracker issues and stage the backport yourself. In
+that case, read on.
+
+
+Creating Backport tracker issues
```

Comment:
What I'm missing here is whether the original master PR needed the `fixes: ....` line before everything below will work??
Especially of those PR's that went into master without a tracker that is hard to fix?

Reply:
No, you can create the master tracker after the master PR is merged. I edited the document to make this more explicit.